apple-pie/styles/style.css

```diff
@@ -0,0 +1,60 @@
+
+body {
+    margin: 0;
+    padding: 0;
+  }
+
+#header {
+ background-image: url("C:\Users\Usuario\Documents\Bootcamp\Week 1\lab-css-recipes-clone\apple-pie\images\apple-pie.jpg");
```

Comment:
never use path that is linked to your computer. If someone downloads your repository and runs your code , they will not have the same path/folder names.. Only use the path looking at the file structure in your working project 

Something like this:

```
 background-image: url(".\images\apple-pie.jpg");
```